### PR TITLE
Improve workshop pages

### LINKED ISSUE #2197: Improve Workshop pages
We need to improve the workshop pages a bit.

Regular workshop pages (under https://mainmatter.com/services/workshops/):

* should we move the mentors below the workshop content (as the content seems more relevant)?

Specific landing pages (e.g. https://rust-telemetry-workshop.mainmatter.com):

* should we move the mentors below the workshop content (as the content seems more relevant)?
* make the call-to-action more prominent (we improved it for the telemetry workshop by adding the button in a few more places but that's kind of a quick fix – would be good to improve that further)
* we need to add pricing information somewhere

### PR DESCRIPTION
**Pull request by yaigrie**
closes #2197

### CODE REVIEW

**Comment by yaigrie** [comment]
@marcoow We'll need to add the pricing info to the Rust and Svelte workshops, I wrote Price TBA added everywhere, where it's missing

**Review by marcoow — COMMENTED** [review]
Should the list of tutors have a bit better separation between the tutors? It seems not super clear where one starts and the other ends?

Bildschirmfoto 2023-11-20 um 09 46 23

Also I'm not sure whether they should be displayed as large on the pages in the workshop catalog (I think on the landing pages it's a different thing)?

On the landing pages, it looks a bit weird that the buttons in the top right and bottom right have different spacing towards the right. Also, I'd remove some of the "book-your-ticket" buttons from the content as it's a lot of buttons currently with the additional one that sticks at the bottom.

Also, let's align the svelte and rust workshop landing pages to look the same

**Review by marcoow — COMMENTED** [review]
I think this is still a bit strange?

> On the landing pages, it looks a bit weird that the buttons in the top right and bottom right have different spacing towards the right. Also, I'd remove some of the "book-your-ticket" buttons from the content as it's a lot of buttons currently with the additional one that sticks at the bottom.

**Comment by yaigrie** [comment]
@marcoow I added a scrolltrigger to make the bottom button only visible after a bit of scrolling 
What do you think?

File: src/rust-application-telemetry-workshop.njk [hunk 1]
```diff
@@ -55,7 +55,8 @@ og:
           </a>
         </div>
         <div class="nav__aside">
-          <tito-button event="mainmatter/telemetry-rust-2023/" class="plausible-event-name=Tickets+List">Book your ticket</tito-button>
+          <tito-button event="mainmatter/telemetry-rust-2023/" class="plausible-event-name=Tickets+List">Book your ticket<div class="line"></div>
+            <span class="price-tag">Price TBA</span></tito-button>
```
Comment (marcoow) [inline src/rust-application-telemetry-workshop.njk]:
```suggestion
            <span class="price-tag">300€</span></tito-button>
```

File: src/rust-application-telemetry-workshop.njk [hunk 2]
```diff
@@ -71,39 +72,24 @@ og:
             <img class="workshop-lp-hero__logo" src="/assets/images/web-based-services-in-rust-workshop/rust-logo.svg" alt="Rust logo"/>
             <p class="workshop-hero__format">Remote Workshop</p>
             <h1 class="workshop-lp-hero__title color-hero__title color-hero__title--full">Telemetry for Rust applications</h1>
-            <div class="color-hero__text color-hero__text--small"><a href="https://everytimezone.com/s/03cf783a">December 12th–13th, 2023 – 14:00 to 18:00 CET</a></div>
+            <div class="color-hero__text color-hero__text--small">
+              <p>
+                <a href="https://everytimezone.com/s/03cf783a">December 12th–13th, 2023 – 14:00 to 18:00 CET</a>
+              </p><br>
+              <p>Take your Rust skills to the next level <strong>for Price TBA</strong>
```
Comment (marcoow) [inline src/rust-application-telemetry-workshop.njk]:
```suggestion
              <p>Take your Rust skills to the next level <strong>for 350€</strong>
```

**Review by marcoow — APPROVED** [review]
🙌

**Comment by yaigrie** [comment]
@marcoow I put the 350€ price on both pages, is this correct?

**Comment by marcoow** [comment]
No, see my comments – one is 350, one is 300

**Comment by yaigrie** [comment]
@marcoow can you tell me again which workshop got which price (sorry, but I think both of your comments were on the rust workshop) :)

**Comment by marcoow** [comment]
ah, sorry – the Rust workshop is 300€, the Svelte one is 350€

**Pull request merged**